Add rail Delay-repay usage example

lddelchev opened the pull request:
Ticket: https://kanbanzone.io/b/YA2t2Uox/c/1075-Rail-Performance-Delay-repay-Usage-Example

Usage example to show basic usage of the `train/station_actual_journeys/{station}.json` endpoint in a delay-repay app.

See on staging: https://developer.transportapi.com/examples-staging?example=train-delay-repay&branch=rail-performance

lddelchev committed:
Add rail Delay-repay usage example

lddelchev committed:
Update dependencies

vdimchev commented on src/examples/bus-journey-patterns/main.js:
```diff
@@ -37,5 +37,6 @@ function tableRowHtml (atcocodeToStopName, journeyPatterns, rowIndex) {
 }
 
 function tableCellHtml (atcocodeToStopName, journeyPattern, rowIndex) {
-  return `<td><b>${atcocodeToStopName[journeyPattern.stops[rowIndex]?.atcocode] || ''}</b> ${journeyPattern.stops[rowIndex]?.atcocode || ''}</td>`
```
What's the purpose of this split? Readibility, linter god?

lddelchev replied:
We were using safety navigation operator (`?.`) which is not supported by the version of node that Codesandbox use and the rendering of the example on their page was broken.

vdimchev replied:
Yes, I was asking exactly because I saw it's still used even after the split.

vdimchev commented on src/examples/train-delay-repay/package.json:
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "places-list",
```
Probably not `places` ;)

lddelchev replied:
Fixed along with other copy-pasted `package.json` files :)

vdimchev reviewed (changes requested):
One small thing.

lddelchev committed:
Remove usage of safety navigation operator

The version of node that Codesandbox uses doesn't support safety navigation operator and the example does not render on their platform

lddelchev committed:
Fix name in examples package.json
